cluster-autoscaler/FAQ.md

```diff
@@ -96,8 +96,13 @@ Cluster Autoscaler decreases the size of the cluster when some nodes are consist
       "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volumes": "volume-1,volume-2,.."
       ```
       and all of the pod's local volumes are listed in the annotation value.
-* Pods that cannot be moved elsewhere due to various constraints (lack of resources, non-matching node selectors or affinity,
-matching anti-affinity, etc)
+* Pods that cannot be moved elsewhere due to scheduling constraints. CA simulates kube-scheduler behavior, and if there's no other node where a given pod can schedule, the pod's node won't be scaled down.
+  * This can be particularly visible if a given workloads' pods are configured to only fit one pod per node on some subset of nodes. Such pods will always block CA from scaling down their nodes, because all
```

Comment:
```suggestion
  * This can be particularly visible if a given workloads' pods are configured to only fit
  one pod per node on some subset of nodes. Such pods will always block CA from scaling
  down their nodes, because all
```

Reply:
nit: Wrap the line